tests/test_chronos_tools.py

```diff
@@ -635,6 +635,47 @@ def test_check_schedule_valid_complete(self):
         assert okay is True
         assert msg == ''
 
+    def test_check_schedule_valid_cron(self):
+        fake_schedule = '* * * * *'
+        chronos_config = chronos_tools.ChronosJobConfig(
+            service='',
+            cluster='',
+            instance='',
+            config_dict={'schedule': fake_schedule},
+            branch_dict={},
+        )
+        okay, msg = chronos_config.check_schedule()
+        assert okay is True
+        assert msg == ''
+
+    def test_check_schedule_invalid_cron_sixth_field(self):
+        fake_schedule = '* * * * * *'
+        chronos_config = chronos_tools.ChronosJobConfig(
+            service='',
+            cluster='',
+            instance='',
+            config_dict={'schedule': fake_schedule},
+            branch_dict={},
+        )
+        okay, msg = chronos_config.check_schedule()
+        assert okay is False
+        assert msg == ('The specified schedule "* * * * * *" '
```

Comment:
Not a blocker to shipping this but I hate that we assert specific error messages in our tests :-(

If you're in a generous mood, fix this up with proper Exception classes and assert that the correct exception is raised instead. The code itself could catch the exception and output the human readable message.